src/parser.js

```diff
@@ -0,0 +1,126 @@
+const parser = require("@babel/parser");
+const traverse = require("@babel/traverse").default;
+const EventEmitter = require("events").EventEmitter;
```

Comment:
I would personally go for an options object with callbacks over an EventEmitter because you don't have to worry about ticks or async.